config/contributors.ts

```diff
@@ -21,21 +22,20 @@ export const Contributor: IContributor[] = [
     name: 'Pratyush Singha',
     role: 'Frontend Developer',
```

Comment:
Hey @pratyushsingha  this seems to be a valid contribution that you made. Why you removed that from your list :)